api/scanner/scanner_user.go

```diff
@@ -169,7 +200,7 @@ func findAlbumsForUser(db *gorm.DB, user *models.User, album_cache *AlbumScanner
 						return err
 					}
 				}
-
+				album.IgnoreFiles = getIgnoreFiles(albumIgnore)
```

Comment:
This could be saved in the `cache` struct, as also mentioned earlier.

Reply:
Will do.